looprpc/version.go

```diff
@@ -0,0 +1,5 @@
+package looprpc
+
+// CurrentProtocolVersion defines the version of the protocol that is currently
+// supported by the loop client.
+const CurrentProtocolVersion = ProtocolVersion_USER_EXPIRY_LOOP_OUT
```

Comment:
Isn't it better to move it to the `loopdb` package, because that are the types that we use internally?

Reply:
The reason it's in the `looprpc` package is because the constant's source is from proto and also there's no circular dependency that is created this way. I'd say it's better to keep it here.

Reply:
Also decided to remove the `ConvertProtocolVersion` function as we only used it to convert the compile time constant. Added a test to keep constants in sync and sane though.